components/resources/ai_chat_ui_strings.grdp

```diff
@@ -264,6 +264,27 @@
   <message name="IDS_CHAT_UI_LEARN_MORE" desc="Link text to learn more about a feature">
     Learn more
   </message>
+  <message name="IDS_CHAT_UI_CLOSE_NOTICE" desc="Label for button which closes a notice">
+    Close this notice
+  </message>
+  <message name="IDS_CHAT_UI_NOTICE_CONVERSATION_HISTORY_TITLE" desc="Title for the notice about conversation history">
+    Conversation history
```

Comment:
This string already exists here (for the title of the conversation history list). Any chance we could share them?

Reply:
I changed it. Either way I think it only gets translated once, but the other key makes some semantic sense here too so why not